pinc/links.inc

```diff
@@ -78,3 +78,46 @@ function private_message_link($proofer_username, $target = "_blank")
 
     return $a;
 }
+
+// -----------------------------------------------------------------------------
+
+function return_to_project_page_link($projectid, $url_suffix = "")
+// Returns a string containing a snippet of HTML to return to a project page.
+// Optional second argument could be page anchor, e.g "#holds"
+// or additional URL parameter, e.g. "&detail_level=4"
```

Comment:
This interface feels error prone since it takes query params or fragments, but also requires the `&`, but you can't send the `?`. Would it make more sense to split them out?
```
function return_to_project_page_link($projectId, $query_param_array, $fragment)
```
Then the caller doesn't need to know about `?`, etc.?

Reply:
Done